cyclonedx/model/component.py

```diff
@@ -962,22 +1047,24 @@ def licenses(self) -> "SortedSet[LicenseChoice]":
     def licenses(self, licenses: Iterable[LicenseChoice]) -> None:
         self._licenses = SortedSet(licenses)
 
-    @property
-    def copyright(self) -> Optional[str]:
+    @property  # type: ignore[misc]
+    @serializable.xml_sequence(11)
+    def copyright_(self) -> Optional[str]:
```

Comment:
breaking change :memo:

Reply:
Avoid pythonic keyword `copyright`

Reply:
i doubt it.
is there any reference?

Reply:
on the data model it makes complete sense to keep with the real wording. 
`copyright` might be reserved - but only on module levels, not on class/object level